packages/google_maps_flutter/google_maps_flutter_ios/ios/Classes/FLTGoogleMapJSONConversions.m

```diff
@@ -141,4 +141,34 @@ + (nullable GMSCameraUpdate *)cameraUpdateFromChannelValue:(NSArray *)channelVal
   }
   return nil;
 }
+
++ (NSArray<GMSStrokeStyle *> *)strokeStylesFromPatterns:(NSArray *)patterns
```

Comment:
The type of `patterns` should specify its lightweight generic.

Reply:
I didn't quite get what you mean, I assume you want me to specify the type somehow? But it's just a json object where the first member is a string (the pattern type) and second a number (the length)

And it's not a dictionary

Reply:
> I didn't quite get what you mean, I assume you want me to specify the type somehow?

Yes, using the lightweight generics syntax, as is done for the return value.

> But it's just a json object where the first member is a string (the pattern type) and second a number (the length)
> 
> And it's not a dictionary

I didn't say it was a dictionary, I just said its type needs to be specified.

> Plus the same has been followed in existing code, for example
> 
> ```
> + (GMSCoordinateBounds *)coordinateBoundsFromLatLongs:(NSArray *)latlongs {
>   return [[GMSCoordinateBounds alloc]
>       initWithCoordinate:[FLTGoogleMapJSONConversions locationFromLatLong:latlongs[0]]
>               coordinate:[FLTGoogleMapJSONConversions locationFromLatLong:latlongs[1]]];
> }
> ```

The existence of older code that [does not follow the current style guide](https://google.github.io/styleguide/objcguide.html#use-lightweight-generics-to-document-contained-types) does not mean that new code doesn't have to follow it.